setup.py

```diff
@@ -18,12 +18,12 @@
 
 from __future__ import print_function, division, absolute_import
 
-import os.path
+import os
```

Comment:
Why?

Reply:
There is some [confusion](https://stackoverflow.com/questions/2724348) around this. I somehow mistakenly recalled that Python 3 encouraged `os` above `os.path`. I'm happy to change it back - more explicit that way.

Reply:
I would change it back, but I'm not too fussed.

Reply:
Changed back.